### DISCUSSION TITLE
Documentation missing `WP_REDIS_USERNAME`

### DISCUSSION THREAD

**Discussion by sebastianthulin** [rhubarbgroup/.github#32] [Support (ROC)] [answered]
Seems to be missing from the documentation. I also tried using ACLs arrays in the WP_REDIS_PASSWORD constant as described. This does not seem to work, as described in github documentation. 

Actions needed to improve the documetation on password protected redis instances: 

- Update the row with password instructions, with a valid format of username/password combination.
- Update documentation with the (working) WP_REDIS_USERNAME constant.

**Comment by tillkruss** [comment, marked as answer]
- `WP_REDIS_PASSWORD` is the way to go, because it's supported with all clients.
- `WP_REDIS_USERNAME` is a relic that shouldn't exist and only works when using Predis as the client.

Feel free to open a Pull Request with the documentation changes: https://github.com/rhubarbgroup/redis-cache/blob/develop/README.md
